File: Courzelo_Angular/src/app/models/Comment/Comment.ts
export class Comment {
    idComment?: string;
    message?: string;
    datecomment?: Date;

    constructor(idComment?: string, message?: string,  datecomment?: Date) {
        this.idComment = idComment;
        this.message = message;
        this.datecomment = datecomment;
      }
}

File: Courzelo-Base/src/main/java/tn/esprit/devflow/courzelo/entity/Comment.java
package tn.esprit.devflow.courzelo.entity;

import groovyjarjarantlr4.v4.runtime.misc.NotNull;
import lombok.AccessLevel;
import lombok.Data;
import lombok.experimental.FieldDefaults;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.DBRef;
import org.springframework.data.mongodb.core.mapping.Document;

import java.util.Date;

@Data
@Document
@FieldDefaults(level= AccessLevel.PRIVATE)

public class Comment {
    @Id
    String idComment;
    String message;
    Date datecomment;
    @DBRef
   Publication publication;
}

